src/clusterfuzz/_internal/bot/tasks/utasks/uworker_io.py

```diff
@@ -349,6 +349,11 @@ def __setattr__(self, attribute, value):
 
     field_descriptor = self.proto.DESCRIPTOR.fields_by_name[attribute]
     if field_descriptor.message_type is None:
+      # Handle the repeated fields of non-composite types.
+      if field_descriptor.label == FieldDescriptor.LABEL_REPEATED:
+        field = getattr(self.proto, attribute)
+        field.extend(value)
```

Comment:
This does not clear the previous field value before extending, which I think will lead to weird behavior:

```py
>>> output.repeated_field = [1, 2]
>>> output.repeated_field = [3, 4]
>>> output.repeated_field
[1, 2, 3, 4]
```

Reply:
+1